Chiazokam/iReporterApp
# #162602934 signin database 162602934

#### What does this PR do?
- implements tests for the sign in route
- implements the sign in route

#### Description of Task to be completed?
- implement tests for the sign in route
- implement the sign in route

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/iReporterApp`
- Enter into the project directory - `cd iReporterApp`
- switch to this branch - `checkout ft-api-signin-database-162602934`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/auth/signin `
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
           "email": "guru@gmail.com",
           "password": "father"
`

#### What are the relevant pivotal tracker stories?
[#162602934](https://www.pivotaltracker.com/story/show/162602934)
